tests/qm-kvm-test/kvm-basic/check_kvm_is_working.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash -x
+
+# shellcheck disable=SC1091
+source ../../e2e/lib/utils
+
+# Build and install the QM sub-package KVM
+install_kvm_subPackage() {
```

Comment:
i tend to skip the  rpmm approach
You can copy the relevant directories for kvm
from 
here 
%{_sysconfdir}/containers/systemd/qm.container.d/qm_dropin_mount_bind_kvm.conf
%{qm_sysconfdir}/containers/systemd/kvm.container

Just copy quadlet file
subsystems/kvm/etc/containers/systemd/kvm.container
And drop in file
etc/containers/systemd/qm.container.d/qm_dropin_mount_bind_kvm.conf

Please use ${TMT_TREE}/etc/containers/systemd/qm.container.d/qm_dropin_mount_bind_kvm.conf
${TMT_TREE}/subsystems/kvm/etc/containers/systemd/kvm.container